content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -30,9 +30,36 @@ To enable debug mode for the Datadog Java Tracer, set the flag `-Ddd.trace.debug
 
 {{< programming-lang lang="python" >}}
 
-To enable debug mode for the Datadog Python Tracer, set the environment variables `DD_TRACE_DEBUG=true` and `DD_CALL_BASIC_CONFIG=true` when using `ddtrace-run`.
+The steps for enabling debug mode Datadog Python Tracer, depends on which version of the tracer your application is using. Choose the scenario that applies:
+
+### Scenario 1: dd-trace-py version 1.3.0 or higher
+
+1. Set the `DD_TRACE_DEBUG` environment variable to `true` to enable debug mode.
+
+2. To route debug logs to a log file, set `DD_TRACE_LOG_FILE` with a filename that tracer logs should be written to, for example, `DD_TRACE_LOG_FILE=ddtrace_logs.log`. 
+   By default, the file size is 15728640 bytes (about 15MB) and one backup log file is created. To increase the default log file size, specify the size in bytes with the `DD_TRACE_LOG_FILE_SIZE_BYTES` setting.
+
+3. To route logs to the console, for **Python 2** applications, configure `logging.basicConfig()` or similar. Logs are automatically sent to the console for **Python 3** applications. 
+
+
+### Scenario 2: dd-trace-py version 1.0.0 to 1.2.0
+
+1. Set these environment variables to enable debug mode:
+    - `DD_TRACE_DEBUG=true`
+    - `DD_CALL_BASIC_CONFIG=true`
+
+2. To route logs to the console, or **Python 2** applications, configure `logging.basicConfig()` or similar. Logs are automatically sent to the console for **Python 3** applications. 
```

Comment:
Since you are explicitly saying to do `DD_CALL_BASIC_CONFIG=true` do they need this step?

Or maybe we shouldn't encourage the use of `DD_CALL_BASIC_CONFIG=true` but keep this step/statement and ask them to do so manually?

Reply:
For **Scenario 2: dd-trace-py version 1.0.0 to 1.2.0**, these two env vars came from our current docs, which tells people to set both right now: https://docs.datadoghq.com/tracing/troubleshooting/tracer_debug_logs/?code-lang=python, which states.

> To enable debug mode for the Datadog Python Tracer, set the environment variables DD_TRACE_DEBUG=true and DD_CALL_BASIC_CONFIG=true when using ddtrace-run.

In 1.x-1.2.x, setting `DD_TRACE_DEBUG=true` doesn't emit logs to the console unless `DD_CALL_BASIC_CONFIG=true` is used or `logging.basicConfig()` gets called.

What do you think of something like this to be clear why we're asking users to enable `DD_CALL_BASIC_CONFIG=true`?

> 1. To enable debug mode: `DD_TRACE_DEBUG=true`
> 2. To route logs to the console, for **Python 2** or **Python 3** applications, configure `logging.basicConfig()` or use `DD_CALL_BASIC_CONFIG=true`.


I mainly want to avoid the case where someone only has `DD_TRACE_DEBUG=true` configured, but don't have debug logs because `logging.basicConfig()` was never called.

Reply:
yeah, I like the new suggestion. encourages `logging.basicConfig()` first. 👍🏻